app/assets/stylesheets/components/_widget.scss

```diff
@@ -0,0 +1,28 @@
+.widget {
+  background: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.